test/rules/assertions/requireParam.js

```diff
@@ -1035,6 +1198,7 @@ export default {
          *
          * @param args Arguments compiled and provided by StephanClient.
          * @param args.options The options as provided by the user, or an empty object if not provided.
+         * @param args.client The options as provided by the user, or an empty object if not provided.
          * @param defaultOptions The default options as provided by the plugin, or an empty object.
```

Comment:
I guess this was just to ensure multiple properties were being checked.

Reply:
Correct, this was to ensure that the test continues to pass now that we're checking each destructured property. If you'd like I can update the property description since it looks like I just copied the description from `args.options`.